### PR TITLE
removing typeahead from o-header for now

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

File: src/js/search.js [hunk 1]
```diff
@@ -19,25 +17,14 @@ const transitionEventName = function(el) {
 
 class Search {
 	static init(headerEl, config = {}) {
-		if (!headerEl) {
-			return;
-		}
+		if (!headerEl) return;
```
Comment (AlbertoElias) [inline src/js/search.js]: We've decided to always go with curly braces in if statements

File: src/js/search.js [hunk 2]
```diff
@@ -19,25 +17,14 @@ const transitionEventName = function(el) {
...
 
 		const form = headerEl.querySelector('[data-o-header-search]');
-		if (!form) {
```
Comment (AlbertoElias) [inline src/js/search.js]: same

File: src/scss/_search.scss [hunk 3]
```diff
@@ -16,7 +16,6 @@
 	height: 45px;
 	padding: 8px 0;
 	box-sizing: border-box;
-	z-index: 5;
```
Comment (AlbertoElias) [inline src/scss/_search.scss]: I think this is still necessary

File: src/js/search.js [hunk 4]
```diff
@@ -27,17 +25,10 @@ class Search {
...
 		const toggle = headerEl.querySelector('[data-o-header-togglable-search]');
 		const input = form.querySelector('input');
-
-		const typeahead = new Typeahead(
-			suggestionsContainer,
-			input,
-			`//${window.location.host}${config.searchDataSrc}`,
-			function() { form.submit(); }
-		);
+		const placeholder = headerEl.querySelector('label');
```
Comment (AlbertoElias) [inline src/js/search.js]: Not necessary

File: src/scss/_nav.scss [hunk 5]
```diff
@@ -19,6 +19,7 @@ $o-header-mobile-width: 5;
 
 	@include oGridRespondTo(M) {
 		position: relative;
+		float: left;
```
Comment (AlbertoElias) [inline src/scss/_nav.scss]: this now breaks

File: src/js/search.js [hunk 6]
```diff
@@ -51,19 +41,24 @@ class Search {
...
 					form.addEventListener(transition, transitionHandler);
 				} else {
 					setTimeout(transitionHandler, 300);
 				}
 
-				typeahead.hide();
+				headerEl.dispatchEvent(new CustomEvent('oHeader.searchToggle', {
+					bubbles: true,
+					detail: {
+						isOpen: getComputedStyle(form, null).getPropertyValue('visibility') === 'visible'
```
Comment (AlbertoElias) [inline src/js/search.js]: this is duplicated

Comment (wheresrhys) [inline src/js/search.js]: IMHO it's a very silly that there is even a transition being used here (even sillier that when there's no transitions supported we add a 300ms delay!). Getting the visibility value once and passing it around would add complexity to the transition end handler, so I don't think it shoudl block this being released

Comment (AlbertoElias) [inline src/js/search.js]: I agree with removing the transition, if next don't want it, great

File: src/scss/_search.scss [hunk 7]
```diff
@@ -16,7 +16,7 @@
 	height: $o-header-bottom-height;
 	padding: 8px 0;
 	box-sizing: border-box;
-	z-index: 5;
+	z-index: 1;
```
Comment (AlbertoElias) [inline src/scss/_search.scss]: have you tried this in multiple browsers?

File: src/js/search.js (hunk 6 above)
Comment (wheresrhys) [inline src/js/search.js]: I'm not saying remove the transition (though I think we should). I just don't want to spend any more time than the absolute minimum working around it

File: src/scss/_search.scss (hunk 7 above)
Comment (wheresrhys) [inline src/scss/_search.scss]: nope. Will do

**Pull request merged**